libs/wire-api/src/Wire/API/User.hs

```diff
@@ -237,25 +240,26 @@ modelUser = Doc.defineModel "User" $ do
 
 instance ToJSON UserProfile where
   toJSON u =
-    object $
-      "id" .= profileId u
-        # "name" .= profileName u
-        # "picture" .= profilePict u
-        # "assets" .= profileAssets u
-        # "accent_id" .= profileAccentId u
-        # "deleted" .= (if profileDeleted u then Just True else Nothing)
-        # "service" .= profileService u
-        # "handle" .= profileHandle u
-        # "locale" .= profileLocale u
-        # "expires_at" .= profileExpire u
-        # "team" .= profileTeam u
-        # "email" .= profileEmail u
-        # []
+    object
+      [ "id" .= _qLocalPart (profileQualifiedId u),
```

Comment:
FUTUREWORK (maybe in another PR): rename `_qLocalPart` to something nicer, e.g. `localId`

Reply:
On second thoughts, this is also used for `Qualified Handle` , so calling it `localId` doesn't make sense. I can't think for a better name than "localPart"

Reply:
Overthinking maybe, but `localPart` is also wrong as it could be a remote thing. So, ideally should be called `unqualified` or `unqualifiedPart`.

Reply:
Anyway, let's continue this discussion in https://github.com/wireapp/wire-server/pull/1290

Reply:
I followed the naming of email adresses there: https://en.wikipedia.org/wiki/Email_address#Local-part